### PR TITLE
Implement FSharp version of Int::sum_v0

### PR DESCRIPTION
**Pull request by jwalter**
Hello, this is my first code contribution to Dark. I'm totally new to both F# and OCaml so I 100% expect there is room for improvement in this PR.

### CODE REVIEW

File: fsharp-backend/src/LibExecution/StdLib/LibInt.fs [hunk 1]
```diff
@@ -275,34 +275,27 @@ let fns : List<BuiltInFn> =
...
+      fn =
+        (function
+        | _, [DList l] ->
+            let ints =
+              List.map
+                (fun i ->
+                  match i with
+                  | DInt it -> it
+                  | _ ->
+                      Errors.throw "Sum expects you to pass a list of ints")
```
Comment (pbiggar) [inline fsharp-backend/src/LibExecution/StdLib/LibInt.fs:291]: Can you use `Errors.argumentWasnt "a list of strings" ...` here?

**Review by pbiggar — CHANGES_REQUESTED** [review]
Awesome, thanks! Just one minor change and then we should be able to ship it.

File: fsharp-backend/src/LibExecution/StdLib/LibInt.fs (hunk 1 above)
Comment (jwalter) [reply fsharp-backend/src/LibExecution/StdLib/LibInt.fs:291]: Absolutely. I assume it should be "a list of ints" though?

Comment (pbiggar) [reply fsharp-backend/src/LibExecution/StdLib/LibInt.fs:291]: oh yeah 🤦

**Comment by pbiggar** [comment]
Wonderful, thank you!

**Pull request merged**